.github/workflows/generate.yml

```diff
@@ -36,9 +36,13 @@ jobs:
     steps:
     - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3
 
-    - name: Compile protobuf
+    - name: Clear out protobuf directory
       run: |
-        make proto
+        rm -rf gen
```

Comment:
nit: can we make this a `make clean` target?

Reply:
yeah I'm doing that... trying to figure out how to get docker to run as the user so clean will work with sudo on local machines

Reply:
all the files are generated with ownership `root:root`

Reply:
:shipit: